### ISSUE TITLE
Suggestion for "condition" items

### ISSUE THREAD

**Issue by JordanRR** [Forelius/fantastic-depths#45]
A nice addition would be something that lets you define some active effect and make it easy to give it to an actor. A simple (?) soluition would be a different "item" type (say a "condition"). It could be basically the same as an item right now (without less data point : ne emcumbrance, no price, etc.), but one that would not be put in the "gear" section but somwhere else (maybe only in the "effect" tab, maybe another, maybe somewhere else.) This way, a simple drag and drop would work. I suppose that it would also be easy to add/delete them with macros etc.

Right now, creating a regular item (with an active effect) works pretty well.

**Comment by Forelius** [comment]
That's funny you mention that, because I did try to do something like what your suggesting and it was going to be a ton of work. My work around is to create a Character who gets assigned all my different needed effects. Then when I need to use an effect I drag drop it from the "Effects Mule Character" to whoever is getting the effect. 
Aside from that the system does define several conditions and I intend to add many more. Those conditions can be applied through the token hud.

**Comment by JordanRR** [comment]
Good idea for a workaround. This way, the character won't have a "gear" item when they suffer a conditon, yet it's still easy to create them beforehand and drag and drop them.

**Comment by Forelius** [comment]
This is now being tracked on issue #74
